src/app.ts

```diff
@@ -105,7 +105,12 @@ export class App {
       }
     });
 
+    this.eventAggregator.subscribe("database.account.signature.successful", () => {
+      this.modalMessage = "Thank you for your patience while we initialize for a few moments...";
```

Comment:
Same text string is used in multiple places -> should add a constant imo.
(Future: think this message is also used in other files, so we should add a concept of "text copies" as a file or directory)